src/components/Badge/__snapshots__/Badge.spec.tsx.snap

```diff
@@ -130,8 +130,8 @@ exports[`Badge should have hover/active styles only when the onClick handler is
   border: 0;
 }
 
-<div
-  class="circuit-0 circuit-1"
+<button
```

Comment:
😍

Reply:
Why is the codecov complaining? 🤔 🙂

Reply:
You can view the [coverage diff](https://codecov.io/gh/sumup-oss/circuit-ui/pull/624/diff?src=pr&el=tree#diff-c3JjL2NvbXBvbmVudHMvQmFkZ2UvQmFkZ2UudHN4) by clicking on the "Impacted files" in the comment above. Look like it's this line in the Badge.tsx file:

<img width="842" alt="Captura de tela 2020-06-23 10 22 28" src="https://user-images.githubusercontent.com/11017722/85379059-9b371e00-b53b-11ea-990f-2e170a07ef09.png">

This condition is made impossible by TypeScript, but I'd leave it in for projects that use JavaScript. So, ignore this coverage drop and merge anyway :)